advanced_tests/test_upgrade.py

```diff
@@ -363,7 +365,8 @@ def upgraded_dcos(dcos_api_session, launcher, setup_workload, onprem_cluster, is
         upgrade_session,
         onprem_cluster,
         dcos_api_session.get_version(),
-        os.environ['TEST_UPGRADE_INSTALLER_URL'])
+        os.environ['TEST_UPGRADE_INSTALLER_URL'],
+        os.environ.get('TEST_UPGRADE_USE_CHECKS'))
```

Comment:
This test is skipped if TEST_UPGRADE_USE_CHECKS is not in environ and we want this argument to be a bool so please do:
```
os.environ['TEST_UPGRADE_USE_CHECKS'] == 'true'
```